src/scss/simple-header/_simple-header.scss

```diff
@@ -0,0 +1,66 @@
+@if $o-header-is-silent == false {
+  .o-header--simple {
+
+    .o-header__nav {
+      display: none;
+    }
+
+    .o-header__top-column--right {
+      .o-header__nav-item {
+        display: none;
+      }
+
+      .o-header__nav-item--myft {
+        display: table-cell;
+        padding-left: 0;
+      }
+    }
+
+
+    @include oGridRespondTo(L) {
+      .o-header__top-logo {
+        display: none;
+      }
+
+      .o-header__nav {
+        display: table;
```

Comment:
because `display: table` is a bit funky maybe `@include oGridRespondTo($until: L) { display: none }` would be less brittle going forward?